lib/http/apps/coin.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+
+const app = express();
+
+app.post(`/issue`, (req, res) => {
+    let name = req.body.name;
+    let symbol = req.body.symbol;
+    let decimals = req.body.decimals;
+
+    console.log(`New coin: ${name} ${symbol} ${decimals}`);
+
+    res.locals.coinsence.Coin.functions.issueCoin(name, symbol, decimals, { gasLimit: 5000000 }).then(transaction => {
+        transaction.wait().then(result => {
```

Comment:
I've been thinking about that for a while... as if we wait for the transaction that means that the HTTP request takes a long time until the transaction gets confirmed. As we can not control that this could lead to a timout. 
I think it is better to assume that the transaction gets mined for now and respond once the transaction is published. 

At some point we need some webhooks or something else to notify the client/humhub that the transaction is now final. 

what are your thoughts on this?

Reply:
we could also add an endpoint to check if a given transaction hash is already confirmed and how many confirmation it has. 
thus the client/humhub could easily poll and check if a transaction is final.

Reply:
by "transaction is published", you mean transaction in pending state, right ?

Reply:
yes. once it is broadcasted to the network. 
basically in `issueCoin().then()`

Reply:
okay good will change that in all endpoints, and will make an endpoint that check if a tx is confirmed.